lib/src/screens/support_other_links/support_other_links_page.dart

```diff
@@ -37,16 +41,16 @@ class SupportOtherLinksPage extends BasePage {
                 if (item is RegularListItem) {
                   return SettingsCellWithArrow(title: item.title, handler: item.handler);
                 }
-
                 if (item is LinkListItem) {
+                  bool lightIcon = false;
```

Comment:
**Minor:**
better naming for readability can be `hasLightIcon`

Reply:
Thanks, change made